lib/util/common.ts

```diff
@@ -62,6 +62,10 @@ export function detectPlatform(
   return null;
 }
 
+export function noLeadingAtSymbol(input: string): string {
+  return input.length && input.startsWith('@') ? input.slice(1) : input;
```

Comment:
```suggestion
  return input.startsWith('@') ? input.slice(1) : input;
```
the other check makes no sense, so just drop it

Reply:
updated and removed